### ISSUE TITLE
Traceback error on Mac through Docker 

### ISSUE THREAD

**Issue by orangeShadow** [codersrank-org/repo_info_extractor#86]
```
Traceback (most recent call last):
  File "src/main.py", line 2, in <module>
    from init import initialize
  File "/app/src/init.py", line 2, in <module>
    import git
  File "/opt/venv/lib/python3.7/site-packages/git/__init__.py", line 38, in <module>
    from git.exc import *                       # @NoMove @IgnorePep8
  File "/opt/venv/lib/python3.7/site-packages/git/exc.py", line 9, in <module>
    from git.compat import UnicodeMixin, safe_decode, string_types
  File "/opt/venv/lib/python3.7/site-packages/git/compat.py", line 16, in <module>
    from gitdb.utils.compat import (
ModuleNotFoundError: No module named 'gitdb.utils.compat'
Finished

```

**Comment by Harmon758** [comment]
See https://github.com/gitpython-developers/GitPython/issues/983.

**Comment by AminSojoudi** [comment]
Same here, @Harmon758  we are using docker image so there is a bug in the image.
I'm using `run-docker.bat <path to the repository>`  with Docker for windows.

**Comment by peti2001** [comment]
It is supposed to be fixed in #99. Can you confirm?
